Gemfile

```
@@ -41,7 +41,7 @@ gem 'sass-rails', '>= 6'
 # An utility first CSS framework
 gem 'tailwindcss-rails', '~> 0.3.0'
 # A framework for building reusable, testable & encapsulated view components in Ruby on Rails
-gem 'view_component', '~> 2.24', require: 'view_component/engine'
+gem 'view_component'
```

```suggestion
gem 'view_component', '~> 2.24'
```
?

we like to use [pessimistic versioning](https://thoughtbot.com/blog/a-healthy-bundle)